### PR TITLE
Create license.txt

### PR DESCRIPTION
**Pull request by nilederg**
We are required to use this license. Change will be merged down through all branches.

### CODE REVIEW

File: license.txt [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+Also add information on how to contact you by electronic and paper mail.
+
+  If the program does terminal interaction, make it output a short
+notice like this when it starts in an interactive mode:
+
+    <program>  Copyright (C) <year>  <name of author>
```
Comment (bend-n) [inline license.txt:655]: this is not how you do a license

**Review by bend-n — CHANGES_REQUESTED** [review]

File: license.txt (hunk 1 above)
Comment (nilederg) [reply license.txt:655]: uh oh

Comment (nilederg) [reply license.txt:655]: i just merged the license from rec, how should we address this?

Comment (bend-n) [reply license.txt:655]: change it ourself
```suggestion
   	Mindustry plague plugin Copyright (C) 2023 Recessive, ACP Contributors
```

Comment (bend-n) [reply license.txt:655]: nvm its fine
you put them in files

**Review by bend-n — APPROVED** [review]

File: license.txt (hunk 1 above)
Comment (bend-n) [reply license.txt:655]: maybe should put them in the files but eh

Comment (nilederg) [reply license.txt:655]: is it really necessary to put in the files?

Comment (bend-n) [reply license.txt:655]: not necessary

**Pull request merged**